### PR TITLE
Add text describing *Description consistency

### LINKED ISSUE #1965: Describe consistency model between *{local|remote}*Description attributes
The spec doesn't enforce a particular consistency model between the localDescription, currentLocalDescription, pendingLocalDescription and their remote counterparts.
These descriptions can potentially change over the time of execution of Javascript, meaning that their values could be observed inconsistently by a script.

Suggestion: Specify that all the six attributes are observed from a single consistent snapshot. (In theory, we could specify that this is observed at the time of script startup, but I think that increases overhead for functions that don't observe these attributes, so I think we should leave the exact time of observation implementation-defined.)

CC @foolip

### PR DESCRIPTION
**Pull request by alvestrand**
Fixes #1965

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/webrtc-pc/pull/2317.html" title="Last updated on Oct 17, 2019, 11:50 AM UTC (3eb3760)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webrtc-pc/2317/53a2dd0...3eb3760.html" title="Last updated on Oct 17, 2019, 11:50 AM UTC (3eb3760)">Diff</a>

### CODE REVIEW

File: webrtc.html [hunk 1]
```diff
@@ -1031,6 +1031,20 @@ <h4>Operation</h4>
...
+        variables <a>[[\CurrentLocalDescription]]</a>,
+        <a>[[\CurrentRemoteDescription]]</a>, <a>[[\PendingLocalDescription]]</a>
+        and <a>[[\PendingRemoteDescription]]</a>. These are only set
+        inside the <code><a>setLocalDescription</a></code>
+        and <code><a>setRemoteDescription</a></code> operations, and
+        modified by the <code><a>addIceCandidate</a></code> operation and
+        the <a>surface a candidate</a> procedure. In each case, all
+        the modifications to all the five variables are completed
+        before the procedures fire any events or invoke any callbacks,
+        so the modifications are made visible as a single event.</p>
```
Comment (henbos) [inline webrtc.html]: Considering we use internal slots, and are clear about when these are updated, I'm not sure there is any ambiguity with these attributes. But it doesn't hurt to have a note about the intent in a place like this.

**Comment by henbos** [comment]
Editors can integrate % rephrasing

File: webrtc.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (alvestrand) [reply webrtc.html]: I'll go with "point in time". Another problem is that I didn't have the right word for the entity that it's made visible to (the Javascript context that owns the variable), but I hope implying what it is will be OK.

Comment (alvestrand) [reply webrtc.html]: I don't think there's any ambiguity either, this text is merely saying what we are aiming for.

**Pull request merged**